### DISCUSSION TITLE
Made this App & Readme Redesign...

### DISCUSSION THREAD

**Discussion by astoko** [PranshulGG/WeatherMaster#217] [Show and tell]
# <img src="https://github.com/user-attachments/assets/e901a33d-7a99-44b6-84a5-d890da570caf" alt="Weather Master" width="450"/>
---
# <img src="https://github.com/user-attachments/assets/aff63340-94f8-49a1-a5e2-01ef2feba61e" alt="Screenshots" width="330"/>
<img src="https://github.com/user-attachments/assets/a8ba87e0-8e65-4b34-ba05-bb3ed88d1784" alt="drawing" width="207"/>
<img src="https://github.com/user-attachments/assets/ad5245a6-d8ca-49c1-9640-1d4711d52fed" alt="drawing" width="207"/>
<img src="https://github.com/user-attachments/assets/95acca40-2813-49b1-9054-066bb430a9d5" alt="drawing" width="207"/>
<img src="https://github.com/user-attachments/assets/8d9408ea-e229-4834-9b7b-ab4f22339a24" alt="drawing" width="207"/>

## <img src="https://github.com/user-attachments/assets/525c00d1-5da1-4416-b2e4-c95e1d61b378" alt="Why did i make these?" width="450"/>

You might wonder and think about why.. no reason the app looks nice and readme too.. just thought it would be nice.. :P
Also yes this **does not count fully redesigned** as.. again it was 3am and just thought it would be fun and spark an fun discussion of what if and maybe actually **someone that knows more than me** does the redesign xD _hehe_

**Comment by astoko** [comment]
Also just you know.. yes i did use hackertype for the "code text" xD

**Reply by PranshulGG** [reply]
Lmao, this actually looks really nice

**Reply by astoko** [reply]
> Lmao, this actually looks really nice 

Thanku thanku :3

**Comment by pdimu** [comment]
I actually like this. If you'd add more screenshots it'd be amazing btw
Would you also add some material design 3 shapes?

**Comment by astoko** [comment]
> I actually like this. If you'd add more screenshots it'd be amazing btw
> Would you also add some material design 3 shapes?

Yeah if I had more I would add but it was actually 3am and my head was on stupid mode so.. XD
Also yeah I was planning to replicate the current version in figma (prototype/design program) and then make the screenshots as well as making an redesigned version :P

Though it may take a little bit of time!

**Reply by pdimu** [reply]
no worries. we don't need them asap. take your time
